pennylane_lightning/src/algorithms/AdjointDiff.cpp

```diff
@@ -16,9 +16,4 @@
 
 // explicit instantiation
 template class Pennylane::Algorithms::AdjointJacobian<float>;
-template class Pennylane::Algorithms::AdjointJacobian<double>;
-
-template class Pennylane::Algorithms::ObsDatum<float>;
-template class Pennylane::Algorithms::ObsDatum<double>;
-template class Pennylane::Algorithms::ObsDatum<std::complex<float>>;
-template class Pennylane::Algorithms::ObsDatum<std::complex<double>>;
```

Comment:
Are these causing issues with compilation? If so, we can declare there existence outside the complaining compilation unit. I think since they are directly defined here I'd like to keep them. Or, we can move them to a separate compilation unit and just link against them (though I understand this may require pulling many threads due to how the templates are currently structured). 

What do you think?

Reply:
Nevermind, I see now you moved everything to a new file.